scripts/tools/nrfconnect/nrfconnect_factory_data.schema

```diff
@@ -23,12 +26,32 @@
             "type": "string",
             "maxLength": 32
         },
+        "vid": {
```

Comment:
Why does it need to be part of the factory data? It doesn't changes across device instances, so can also be the part of firmware, right?

Reply:
I think the reason is that we can update firmware during DFU and change the vid/pid that are saved in firmware. That will break compatibility with DAC that is in the factory data, so we wanted to keep them consistent this way.

Reply:
Ok, I think it is rather unusual scenario and also `GetVendorXX/GetProductXX` methods are still part of the `ConfigurationManager` interface, so if you would like to make them part of the factory data, you need to think which interface they should be moved to so that we can implement the factory data accessor that is independent from the configuration manager implementation.

Reply:
Yes, of course, aligning API was part of a plan, but we wanted to do that in separate PR with factory data accessor, as this one just adds script that will not be used in any case yet.